### ISSUE TITLE
calculalte accuracy or another metric for all test set

### ISSUE THREAD

**Issue by hosein-jamshidian** [hosein-jamshidian/Persian-Traffic-Sign-Recognition#1]
i think this is better you add somthing like accuracy or f1-score for test set  .thank you

**Comment by hosein-jamshidian** [comment]
i think another important point is that why the validation accuracy is better than training accuracy !
